full/Dockerfile

```diff
@@ -8,6 +8,15 @@ RUN yes | unminimize \
         build-essential \
         htop \
         jq \
+        hub=2.7.0~ds1-1 \
+        zsh=5.5.1-1ubuntu3 \
+        tmux=2.8-3 \
+        fzf=0.17.5-2 \
+        mtr=0.92-2 \
+        ripgrep=0.10.0-2 \
+        neovim=0.3.4-1ubuntu0.19.04.1 \
```

Comment:
We have theia

Reply:
See #76

Reply:
I like the idea of having both `vim` and `neovim` in our images. It's just an extra quality-of-life thing for developers who usually use `neovim`, and discover it's available in Gitpod without extra effort.

(But again, only if its installed size is insignificant. If it's significant, we'd have to really evaluate usefulness vs cost.)